### ISSUE TITLE
communities lis when tagggin content on GCcollab shows GCconnex communities

### ISSUE THREAD

**Issue by LemieuxGen** [gctools-outilsgc/gccollab#214]
When creating content on GCcollab, and are asked to tag the content with communities, the dropdown list for the communities shows communities from GCconnex, which are (mostly) relevant to GC public servants. These are also different from the list of communities from the main navigation menu.

![gccollab communities](https://user-images.githubusercontent.com/15378977/30934989-49b3638e-a39d-11e7-8c0d-9917d866c292.JPG)

@JOutram77 do you know which list of communities was supposed to be used on GCcollab?

**Comment by JOutram77** [comment]
I was just made aware that this communities function was ported from GCconnex. This was something done while I was on vacation. I don't believe that these are appropriate for the GCcollab but do not have a list of communities that should be used here. This is part of a larger 'communities' discussion for Collab that needs to take place. This is only one piece of a larger effort.

**Comment by ToferC** [comment]
Best bet is probably to pull categories from the open data portal. More
thought would be better, but we can get the data now and assess later.

**Comment by markwooff** [comment]
It would be ideal of the list of Communities on GCcollab were congruent with the Audiences that are suggested when creating new content. I can set the audiences to be `Gov21`, `Human Health`, `Jobs and Skill Development`, and `Tech` but I think this list needs to be fleshed out a bit more to create a final set of Communities.

**Comment by ToferC** [comment]
Agreed @wooff.

**Comment by markwooff** [comment]
@JOutram77 have any discussions taken place about a final list of communities for GCcollab? If not, should we schedule something?

**Comment by JOutram77** [comment]
@markwooff no, I've been focussed on on-boarding municipalities. I'll schedule something for early next week.
